docs/notebooks/using_mclip.ipynb

```diff
@@ -0,0 +1,316 @@
+   "source": [
+    "## Saving\n",
+    "\n",
+    "After the run has finished successfully, you can download the tuned model on your local machine:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0476c03f-838a-4589-835c-60d1b7f3f893",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "artifact = run.save_artifact('m-clip-model')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "baabd6be-8660-47cc-a48d-feb43d0a507b",
+   "metadata": {},
+   "source": [
+    "## Inference\n",
+    "\n",
+    "Now you saved the `artifact` into your host machine,\n",
+    "let's use the fine-tuned model to encode a new `Document`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fe43402f-4191-4343-905c-c75c64694662",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "text_da = DocumentArray([Document(text='setwas Text zum Codieren')])\n",
+    "image_da = DocumentArray([Document(uri='https://upload.wikimedia.org/wikipedia/commons/4/4e/Single_apple.png')])\n",
+    "\n",
+    "clip_text_encoder = finetuner.get_model(artifact=artifact, select_model='clip-text')\n",
+    "clip_image_encoder = finetuner.get_model(artifact=artifact, select_model='clip-vision')\n",
+    "\n",
+    "finetuner.encode(model=clip_text_encoder, data=text_da)\n",
+    "finetuner.encode(model=clip_image_encoder, data=image_da)\n",
+    "\n",
+    "print(text_da.embeddings.shape)\n",
+    "print(image_da.embeddings.shape)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ff2e7818-bf11-4179-a34d-d7b790b0db12",
+   "metadata": {},
+   "source": [
+    "```bash\n",
+    "(1, 512)\n",
+    "(1, 512)\n",
+    "```\n",
+    "\n",
+    "```{admonition} what is select_model?\n",
+    "When fine-tuning CLIP, we are fine-tuning the CLIPVisionEncoder and CLIPTextEncoder in parallel.\n",
+    "The artifact contains two models: `clip-vision` and `clip-text`.\n",
+    "The parameter `select_model` tells finetuner which model to use for inference, in the above example,\n",
+    "we use `clip-text` to encode a Document with text content.\n",
+    "```\n",
+    "\n",
+    "```{admonition} Inference with ONNX\n",
+    "In case you set `to_onnx=True` when calling `finetuner.fit` function,\n",
+    "please use `model = finetuner.get_model(artifact, is_onnx=True)`\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "99c3f43f-01e4-4f38-8480-1a0dde3460bb",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.10"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```

Comment:
![3BFEB298-846C-4A3C-A18F-19AE22CEBCA6](https://user-images.githubusercontent.com/9794489/203245653-e62248bd-a7c4-4192-adb2-481239cd7911.png)
we need to login to the team account and push the da to the cloud

Reply:
![6FED2752-43F4-4361-9B42-A9F175435A4F](https://user-images.githubusercontent.com/9794489/203245772-5671625f-7a64-4a1e-b95a-3686c5c9cb85.png)
which model, can you elaberate?

Reply:
![8E026CAD-060A-441E-9874-A67C6E1FC8ED](https://user-images.githubusercontent.com/9794489/203245954-58345aeb-f063-49b7-b562-0e1cf706aa7d.png)
always use cuda

Reply:
The method of pulling data and the use of 'cpu' were for testing purposes, updated now